Makefile

```diff
@@ -0,0 +1,32 @@
+CFLAGS:=--std=c++1z -O2 -fPIC -DGNU_GCC -D_GNU_SOURCE -D_GLIBCXX_USE_CXX11_ABI=0 -DBOOST_DISABLE_ASSERTS
+LDFLAGS:=-Wl,-E -Wl,--hash-style=gnu
+ROOTLIBS:=-lCore -lRIO -lNet -lHist -lMatrix -lThread -lTree -lMathCore -lGpad -lGraf3d -lGraf -lPhysics -lPostscript -lASImage
+INCLUDE=-I. -I$(ROOT_ROOT)/include -I$(BOOST_ROOT)/include
+LIBDIR=-L./build/lib -L$(shell echo $(LD_LIBRARY_PATH) | sed 's|:| -L|g')
+OTHERLIBS:=-ldl -ljpeg -lpng
+
+all: lib/renderplugin.so bin/render lib/libDQMRenderPlugins.so
+	@echo All build
+
+objs/src/%.cc.o: src/%.cc
+	@mkdir -p $(@D)
+	g++ -c $(CFLAGS) $(INCLUDE) -o $@ $<
+
+objs/plugins/%.cc.o: plugins/%.cc
+	@mkdir -p $(@D)
+	g++ -c $(CFLAGS) -O3 $(INCLUDE) -o $@ $<
+
+lib/renderplugin.so: objs/src/DQMRenderPlugin.cc.o
```

Comment:
by the way, why you named it renderplugin.so and why not librenderplugin.so?

Reply:
can we rename this to librenderplugin.so ? currently rpm packaging system fails to treat it as library and complains that render executable needs it but no one provide this library. renaming it to librenderplugin.so will solve this issue